src/content/blog/2024-08-22-the-ugrc-api-address-location-lane.mdx

```diff
@@ -0,0 +1,45 @@
+
+## Road centerlines versus address points
+
+Road centerlines and address points are two different datasets that work together to provide the best address location match rates. By default, both datasets are used to provide the solution and address points have priority in the case of a tie. The UGRC API provides which dataset solved your request in the response and you are able to specify which data participates in the match using the `locator` optional input. This is important since the road centerlines are used to determine the location of an address along a road segment, while the address points are used to determine the exact location of an address. By using both datasets together, you will get the best match rate our data can provide. But a match on the road centerlines has a different meaning than a match on the address points. But if you are zooming, routing, extracting data, or looking for a generalized location for analysis a match on the road centerlines is a great option.
+
+<Image
+  src={centerlineImage}
+  width={800}
+  alt="A line indicating fiction street with a circle at the 25% location along the segment"
+/>
+
+To understand why, lets break down what happens with a match on the road centerlines with an example. Consider the input street address is `125 South Fiction Street`. The road centerline data has a segment representing Fiction Street with attributes defining the address range for that segment e.g. 100-200. This guarantees that any address on this segment will have numbers between 100 and 200. Since 125 falls within the address range of the segment, the road centerline address locator will provide a match that is 25% along the line segment in the positive direction. However, the exact location of the address along the segment is not known; It may not exist in terms of a structure, mail delivery, or the availability of 911 services. **If you are solving or validating for mail delivery, 911 response, voter registration, etc., this is not an accurate solution.** You may need to evaluate services provided by the [USPS](https://www.usps.com/) or [Smarty](https://smarty.com/) to validate mail delivery.
+
+Address location matches solved by the address points do represent a geographic location that has been assigned a US Postal Service (USPS) address by the local address authority (i.e., county or municipality) but does not necessarily receive mail. For example, mail may be delivered to a post office box.
```

Comment:
```suggestion
Address location matches solved by the address points represent a geographic location that has been assigned a civic address by the local address authority (i.e., county or municipality) but does not necessarily receive mail. For example, mail may be delivered to a post office box.
```

Reply:
This was taken directly from our address points product page. Do you want to update that to match these suggestions?

Reply:
@ZachBeck what are your thoughts?

Reply:
I think this makes more sense than mentioning the USPS like on the address page.

Reply:
@gregbunce or @ZachBeck do you mind synchronizing these changes with the address product page and make sure it's also reflected in @miriamseely's metadata work?